{"uuid":"e2806e70-ac29-443d-a6ff-78bd968daf70","historyId":"acb74640d786c2d5477e53fae93ad4c6","fullName":"Pays.posfinaltesting.LoginPasscodeTest.Passcode","labels":[{"name":"package","value":"Pays.posfinaltesting.LoginPasscodeTest"},{"name":"testClass","value":"Pays.posfinaltesting.LoginPasscodeTest"},{"name":"testMethod","value":"Passcode"},{"name":"parentSuite","value":"Suite"},{"name":"suite","value":"Login_Passcode"},{"name":"subSuite","value":"Pays.posfinaltesting.LoginPasscodeTest"},{"name":"host","value":"Pratik"},{"name":"thread","value":"24228@Pratik.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"feature","value":"Pass Feature"},{"name":"story","value":"Valid login test"},{"name":"epic","value":"EP001"},{"name":"epic","value":"Login Managment"},{"name":"feature","value":"Login And Passcode"},{"name":"story","value":"Login "},{"name":"severity","value":"critical"}],"links":[],"name":"Passcode","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Appium HTTP server is not listening at http://127.0.0.1:4723/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.\nNode.js executable path: C:\\Program Files\\nodejs\\node.exe\nArguments: [C:\\Users\\prati\\AppData\\Roaming\\npm\\node_modules\\appium\\build\\lib\\main.js, --port, 4723, --address, 127.0.0.1]\nServer log: \u001B[38;5;112m[Appium]\u001B[0m Welcome to Appium v2.15.0\r\n\u001B[38;5;112m[Appium]\u001B[0m Non-default server args:\r\n\u001B[38;5;112m[Appium]\u001B[0m { address: \u001B[32m'127.0.0.1'\u001B[39m }\r\n\u001B[38;5;112m[Appium]\u001B[0m The autodetected Appium home path: C:\\Users\\prati\\.appium\r\n\u001B[38;5;112m[Appium]\u001B[0m Attempting to load driver uiautomator2...\r\n\u001B[38;5;112m[Appium]\u001B[0m Requiring driver at C:\\Users\\prati\\.appium\\node_modules\\appium-uiautomator2-driver\\build\\index.js\r\n\u001B[38;5;112m[Appium]\u001B[0m AndroidUiautomator2Driver has been successfully loaded in 0.849s\r\n\u001B[38;5;86m[HTTP]\u001B[0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.\r\n\u001B[38;5;112m[Appium]\u001B[0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4723\r\n\u001B[38;5;112m[Appium]\u001B[0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4723\n    at Server.setupListenHandle [as _listen2] (node:net:1937:16)\n    at listenInCluster (node:net:1994:12)\n    at node:net:2203:7\n    at processTicksAndRejections (node:internal/process/task_queues:90:21)\r\n","trace":"io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: Appium HTTP server is not listening at http://127.0.0.1:4723/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.\nNode.js executable path: C:\\Program Files\\nodejs\\node.exe\nArguments: [C:\\Users\\prati\\AppData\\Roaming\\npm\\node_modules\\appium\\build\\lib\\main.js, --port, 4723, --address, 127.0.0.1]\nServer log: \u001B[38;5;112m[Appium]\u001B[0m Welcome to Appium v2.15.0\r\n\u001B[38;5;112m[Appium]\u001B[0m Non-default server args:\r\n\u001B[38;5;112m[Appium]\u001B[0m { address: \u001B[32m'127.0.0.1'\u001B[39m }\r\n\u001B[38;5;112m[Appium]\u001B[0m The autodetected Appium home path: C:\\Users\\prati\\.appium\r\n\u001B[38;5;112m[Appium]\u001B[0m Attempting to load driver uiautomator2...\r\n\u001B[38;5;112m[Appium]\u001B[0m Requiring driver at C:\\Users\\prati\\.appium\\node_modules\\appium-uiautomator2-driver\\build\\index.js\r\n\u001B[38;5;112m[Appium]\u001B[0m AndroidUiautomator2Driver has been successfully loaded in 0.849s\r\n\u001B[38;5;86m[HTTP]\u001B[0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.\r\n\u001B[38;5;112m[Appium]\u001B[0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4723\r\n\u001B[38;5;112m[Appium]\u001B[0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4723\n    at Server.setupListenHandle [as _listen2] (node:net:1937:16)\n    at listenInCluster (node:net:1994:12)\n    at node:net:2203:7\n    at processTicksAndRejections (node:internal/process/task_queues:90:21)\r\n\r\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:192)\r\n\tat Pays.posfinaltesting.BaseTest.ConfigureAppium(BaseTest.java:31)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)\r\nCaused by: io.appium.java_client.service.local.AppiumServerAvailabilityChecker$ConnectionTimeout: ConnectionTimeout\r\n\tat io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:72)\r\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:144)\r\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:185)\r\n\t... 25 more\r\nCaused by: java.net.ConnectException: Connection refused: getsockopt\r\n\tat java.base/sun.nio.ch.Net.pollConnect(Native Method)\r\n\tat java.base/sun.nio.ch.Net.pollConnectNow(Net.java:682)\r\n\tat java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)\r\n\tat java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592)\r\n\tat java.base/java.net.Socket.connect(Socket.java:751)\r\n\tat java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:178)\r\n\tat java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:531)\r\n\tat java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:636)\r\n\tat java.base/sun.net.www.http.HttpClient.<init>(HttpClient.java:280)\r\n\tat java.base/sun.net.www.http.HttpClient.New(HttpClient.java:386)\r\n\tat java.base/sun.net.www.http.HttpClient.New(HttpClient.java:408)\r\n\tat java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1319)\r\n\tat java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1252)\r\n\tat java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1138)\r\n\tat java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1067)\r\n\tat io.appium.java_client.service.local.AppiumServerAvailabilityChecker.connectToUrl(AppiumServerAvailabilityChecker.java:79)\r\n\tat io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:61)\r\n\t... 27 more\r\n"},"stage":"finished","description":"PassCode functionality with valid credentials","steps":[],"attachments":[],"parameters":[],"start":1742483011779,"stop":1742483011781}